debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogFile.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.logminer;
+
+import java.util.Objects;
+
+/**
+ * Represents a redo or archive log in Oracle.
+ *
+ * @author Chris Cranford
+ */
+public class LogFile {
+
+    private static final String CURRENT = "CURRENT";
+
+    private final String fileName;
+    private final Scn firstScn;
+    private final Scn nextScn;
+    private final String status;
```

Comment:
Should this be an enum?

Reply:
Looking back, I actually think we can simply resolve `"CURRENT".equals(rs.getString(...))` before we create the `LogFile` object and pass true/false here for this.  We only care if its set to `CURRENT` and nothing else, so that should suffice.